ux.symfony.com/code_snippets/_swup.html.twig

```diff
@@ -1,8 +1,6 @@
-{# initialize the swup controller, then render normal links! #}
-<div
-    {{ stimulus_controller('symfony/ux-swup/swup') }}
-    id="swup"
->
+{# you only need to add the call to the Stimilus controller;
+   swup doesn't require any other changes in your template #}
+<div {{ stimulus_controller('symfony/ux-swup/swup') }}>
```

Comment:
How is the `symfony/ux-swup/swup` string formed?

* I guess `symfony/` is hardcoded
* `ux-swup/` not sure where it comes from
* `/swup` is the name of the controller action?

Reply:
It's all "hardcoded". Or, to say it better, when you install the package, the package provides you with a Stimulus controller. And, it gets to choose what name to use for that controller.

In some packages, this info is a bit hidden - e.g. with chart.js, you use `{{ render_chart(chart) }}` and that outputs the element with the correct `data-controller` for you.

Knowing this, what would be your ideal way for this to look/work? I agree that this string looks a bit random.